care/users/reset_password_views.py

```diff
@@ -49,7 +49,36 @@ class ResetPasswordCheck(GenericAPIView):
 
     permission_classes = ()
 
-    @extend_schema(tags=["auth", "users"])
+    @extend_schema(
+        tags=('users', 'auth'),
+
+
+        examples=[
+            OpenApiExample(
+
+                name='Password Reset Check Example',
+                summary='Password Reset Check',
+                description='Provide your unique token',
+                value={
+                    "token": "your token"
+                },
```

Comment:
this needs to be defined as parameters.
https://drf-spectacular.readthedocs.io/en/latest/customization.html#step-2-extend-schema

do the same for other methods

Reply:
Do we have to define the QuerySerializer or it by default provides us.